stackrox/stackrox
# ROX-18008: Delete non-postgres code in Entity UI

## Description

> deprecating our Postgres feature flag

Van, can you please look carefully at changes to GraphQL queries?

### Analysis

Find in Files decrease from 36 results in 30 files to 24 results in 18 files

| Folder                        | results | files |
| :---                          |    ---: |  ---: |
| Components/workflow           |       1 |     1 |
| Containers/VulnMgmt/Dashboard |       1 |     1 |
| Containers/VulnMgmt/Entity    |      12 |    12 |
| Containers/VulnMgmt/List      |      10 |     8 |
| Containers/VulnMgmt/widgets   |      11 |     7 |
| types                         |       1 |     1 |
|                               |      36 |    30 |

### Residue

1. Fix inconsistent case in phrase under **Findings**
2. Investigate effort to replace **the whatever you are looking for cannot be found** with 404 page for obsolete entity types: components, cves, and policies.
3. Investigate whether VulnMgmtPolicyQueryUtil is still relevant.
4. Verify whether deletions from List folder remove last reference:
    * VULN_CVE_LIST_FRAGMENT

## Checklist

## Testing Performed

1. `yarn lint` in ui
2. `yarn build` in ui
    * `wc build/static/js/*.js`
        branch - master: -13771 = 11771992 - 11785763
3. `yarn start` in ui

### Manual testing

1. Visit /main/vulnerability-management/clusters and click a row
    * See 3 tabs under **Cluster Findings**
        ![clusters](https://github.com/stackrox/stackrox/assets/11862657/736ec034-5af0-4271-8402-e9b3aae9fa8a)

2. Visit /main/vulnerability-management/deployments and click a row that has fixable CVEs
    * See fixable image CVEs in table under **Deployment Findings**
        ![deployments](https://github.com/stackrox/stackrox/assets/11862657/fcb66b02-2393-4171-8e01-15711ebedb46)

3. Visit /main/vulnerability-management/images and click a row that has fixable CVEs
    * See fixable image CVEs in table under **Image Findings**
        ![images](https://github.com/stackrox/stackrox/assets/11862657/191bf112-def7-450d-ab80-b9dabe0b5f2e)

4. Visit /main/vulnerability-management/namespaces and click a row that has fixable CVEs
    * See fixable image CVEs in table under **Namespace Findings**
        ![namespaces](https://github.com/stackrox/stackrox/assets/11862657/1551af98-9873-469b-b6bb-e88bdbdefd16)

5. Visit /main/vulnerability-management/nodes and click a row that has fixable CVEs
    * See fixable node CVEs in table under **Node Findings**
        ![nodes](https://github.com/stackrox/stackrox/assets/11862657/6379d2c2-354f-43e4-b20b-c35f6b89daa6)

### Integration testing

1. `yarn cypress-open` in ui/apps/platform
    * vulnmanagement/clusterCves.test.js
    * vulnmanagement/clusters.test.js
    * vulnmanagement/deployments.test.js
    * vulnmanagement/imageComponents.test.js
    * vulnmanagement/imageCves.test.js
    * vulnmanagement/images.test.js
    * vulnmanagement/namespaces.test.js
    * vulnmanagement/nodeComponents.test.js
    * vulnmanagement/nodeCves.test.js
    * vulnmanagement/nodes.test.js
